paasta_tools/tron_tools.py

```diff
@@ -1046,6 +1051,9 @@ def format_tron_action_dict(action_config: TronActionConfig):
                 limit=63,
                 suffix=4,
             ),
+            "tron.yelp.com/idempotent-action": str(
```

Comment:
i'd personally prefer
```suggestion
            "tron.yelp.com/is-idempontent": str(
```
...but i also can't really justify why, so up to you :)

Reply:
If you do this don't commit the suggestion, there is a typo in "idempontent" lol